src/js/components/molecules/sys-nav/sys-nav.vue.html

```diff
@@ -6,6 +6,7 @@
                 <span class="sys-nav__label sys-nav__label--hidden-sm">{{ orderNowLabel }}</span>
             </base-link>
         </li>
+        <!--
```

Comment:
assuming that this is commented out because we will shortly uncomment it once the search issues are resolved.